tests/foreman/ui/test_org.py

```diff
@@ -442,37 +451,38 @@ def test_remove_hostgroup(self, host_grp_name):
         """
         strategy, value = common_locators['entity_select']
         strategy1, value1 = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        host_grp = entities.HostGroup(name=host_grp_name).create()
-        self.assertEqual(host_grp.name, host_grp_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name, hostgroups=[host_grp_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_hostgrps'])
-            element = session.nav.wait_until_element((strategy1,
-                                                      value1 % host_grp_name))
-            # Item is listed in 'Selected Items' list and not 'All Items' list.
-            self.assertIsNotNone(element)
-            self.org.update(org_name, hostgroups=[host_grp_name],
-                            new_hostgroups=None)
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_hostgrps'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % host_grp_name))
-            # Item is listed in 'All Items' list and not 'Selected Items' list.
-            self.assertIsNotNone(element)
+            for host_grp_name in generate_strings_list():
```

Comment:
Same comment as before about creating only one organization and then adding/removing host groups. Not a blocker for this PR.